waspc/data/Generator/templates/server/src/routes/index.js

```diff
@@ -16,5 +17,6 @@ router.get('/', function (req, res, next) {
 router.use('/auth', auth)
 {=/ isAuthEnabled =}
 router.use('/{= operationsRouteInRootRouter =}', operations)
+router.use(apis)
```

Comment:
By putting at the end, it should not be allowed to interfere with any of our routes defined before it. Route setting in Express appears to be first set wins.

Reply:
Oh intersting, ok! Maybe we could add a comment here explaining that? if you think it is redundant, ignore this comment.

Reply:
Added comment 👍🏻